package/network/utils/iproute2/patches/999-honor-copts.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
++++ b/Makefile
+@@ -49,7 +49,7 @@ CCOPTS = -O2 $(EXTRA_CCOPTS)
+ WFLAGS := -Wall -Wstrict-prototypes  -Wmissing-prototypes
+ WFLAGS += -Wmissing-declarations -Wold-style-definition -Wformat=2
+ 
+-CFLAGS := $(WFLAGS) $(CCOPTS) -I../include $(DEFINES) $(CFLAGS)
++CFLAGS := $(WFLAGS) -I../include $(EXTRA_CCOPTS) $(DEFINES) $(CFLAGS)
```

Comment:
Actually I think the entire 999 patch is bogus since 110-extra-ccopts does
CCOPTS = -O2 $(EXTRA_CCOPTS)

So CCOPTS already has $EXTRA_CCOPTS.